airbyte-integrations/connectors/destination-bigquery-denormalized/metadata.yaml

```diff
@@ -2,18 +2,23 @@ data:
   connectorSubtype: database
   connectorType: destination
   definitionId: 079d5540-f236-4294-ba7c-ade8fd918496
-  dockerImageTag: 1.5.3
+  dockerImageTag: 2.0.0
   dockerRepository: airbyte/destination-bigquery-denormalized
   githubIssueLabel: destination-bigquery-denormalized
   icon: bigquery.svg
   license: ELv2
   name: BigQuery (denormalized typed struct)
   registries:
     cloud:
-      enabled: true
+      enabled: false
     oss:
-      enabled: true
-  releaseStage: beta
+      enabled: false
```

Comment:
@bnchrch reading https://www.notion.so/Hiding-or-removing-a-connector-from-Cloud-0839d6c26445415d8aab5643a84db75e, I think that this collection of metadata will:
* Present the breaking change banner for those users who currently use the connector
* Disallow folks from making new connectors with this destination
... is that right?

Reply:
It will not do this 

It will remove this destination from both registries - therefore the new version with the "breaking change" will never hit cloud. For OSS users who already have bq-denormalized, this would work the same way. This only removes it from a fresh instance of OSS.

If we want to hide it from the list of connectors returned when creating new connections, the canon way to do that is with the feature flag (dont have a link rn but see our backlog item with 2 points for removing frontend excluded connectors to this backend feature flag)

Reply:
@erohmensing is right on the mechanics here.

A caveat is the desired flow is here for cloud. By
1. In one PR adding the breaking change to the metadata
2. In a subsequent PR, setting registry enabled values to false
3. In a final PR removing the connector folder / metadata all together

You will have
1. Added a breaking change banner for all existing users
2. Prevented customers from becoming new users of this connector
3. Removed the connector from the codebase

The piece that is missing is oss where there is likely few users who will have updated their connector list in quick succession at the right moments between step 1/2 and 2/3

Im not sure thats too avoidable at the moment though, and maybe have to be part of the experience until we can find a better way to allow OSS users to import registry updates automatically.

Reply:
I still don't think you've removed the connector from the codebase until you've tombstoned it in the prod db (for cloud)